src/System Application/Test/Rest Client/src/RequestMessageTests.Codeunit.al

```diff
@@ -145,12 +145,86 @@ codeunit 134972 "Request Message Tests"
 
         // [THEN] The request message is initialized correctly
         HttpRequestMessage.GetHeaders(ContentHeaders);
-        Assert.AreEqual(ContentHeaders.Contains('X-Custom-Header'), true, 'The custom header is missing.');
+        Assert.IsTrue(ContentHeaders.Contains('X-Custom-Header'), 'The custom header is missing.');
 
         ContentHeaders.GetValues('X-Custom-Header', ContentHeaderValues);
-        Assert.AreEqual(ContentHeaderValues.Get(1), 'My Request Header', 'The custom header value is not correct.');
+        Assert.AreEqual('My Request Header', ContentHeaderValues.Get(1), 'The custom header value is not correct.');
     end;
 
+    [Test]
+    [NonDebuggable]
+    procedure TestAddSecretRequestHeader()
+    var
+        ALHttpRequestMessage: Codeunit "Http Request Message";
```

Comment:
Did you intent to use the prefix `AL` for the AL HttpRequestMessage Type?
Then it's currently the other way.

It's the same on other test functions